### PR TITLE
Add some new icons

### LINKED ISSUE #89: Remove the Power Plug icon
#### Describe the bug
Need to remove the Power Plug icon. It is already available through Material icons.

#### What is the expected behavior?
No duplicates.

### PR DESCRIPTION
**Pull request by joebochill**
Fixes #89 .
For PXBLUE-1570

#### Changes proposed in this Pull Request:
- Remove the power plug icon (it already exists in material)
- Add Internet, Account Settings, and Lock Reset icons

### CODE REVIEW

**Comment by huayunh** [comment]
Can you also double-check that those icons me and Kyle inner-sourced the other day are made by Eric or Material Design? We attributed all of them to Eric but that’s only bc Eric passed them to me

**Comment by joebochill** [comment]
> Can you also double-check that those icons me and Kyle inner-sourced the other day are made by Eric or Material Design? We attributed all of them to Eric but that’s only bc Eric passed them to me

I have re-attributed them in the meta file. Looks like the only one we did in-house was the dimmer.

**Comment by huayunh** [comment]
Ah right that was the exception. Can you put that under my name instead?

File: design/General/account_settings.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><title>Untitled-1</title><path d="M9.6,3.32a3.86,3.86,0,1,0,3.86,3.85A3.85,3.85,0,0,0,9.6,3.32M16.35,11a.26.26,0,0,0-.25.21l-.18,1.27a4.63,4.63,0,0,0-.82.45l-1.2-.48a.3.3,0,0,0-.3.13l-1,1.66a.24.24,0,0,0,.06.31l1,.79a3.94,3.94,0,0,0,0,1l-1,.79a.23.23,0,0,0-.06.3l1,1.67c.06.13.19.13.3.13l1.2-.49a3.85,3.85,0,0,0,.82.46l.18,1.27a.24.24,0,0,0,.25.2h1.93a.24.24,0,0,0,.23-.2l.18-1.27a5,5,0,0,0,.81-.46l1.19.49c.12,0,.25,0,.32-.13l1-1.67a.23.23,0,0,0-.06-.3l-1-.79a4,4,0,0,0,0-.49,2.67,2.67,0,0,0,0-.48l1-.79a.25.25,0,0,0,.06-.31l-1-1.66c-.06-.13-.19-.13-.31-.13L19.5,13a4.07,4.07,0,0,0-.82-.45l-.18-1.27a.23.23,0,0,0-.22-.21H16.46M9.71,13C5.45,13,2,14.7,2,16.83v1.92h9.33a6.65,6.65,0,0,1,0-5.69A13.56,13.56,0,0,0,9.71,13m7.6,1.43a1.45,1.45,0,1,1,0,2.89,1.45,1.45,0,0,1,0-2.89Z"/></svg>
```
Comment (huayunh) [inline design/General/account_settings.svg:1]: If there is no issue with that `<title>Untitled-1</title>` then I am happy with this

**Review by huayunh — APPROVED** [review]

**Pull request merged**